frontend/src/app/interceptors/fake-backend.interceptor.ts

```diff
@@ -108,6 +108,97 @@ export class FakeBackendInterceptor implements HttpInterceptor {
         status: 200,
       }));
     }
+
+    if (request.url.includes('/distribution/age-sex/cases')) {
+
+      return of(new HttpResponse({
+        body: {
+          items: request.params.getAll('area-id').map(area =>
+            ({
+              areaId: area,
+              areaName: 'A-' + area,
+              data: getAgeRanges.map((range, idx) =>
+                ({
+                  ageIntervalId: idx,
+                  ageInterval: range
+                  ,
+                  values: [
+                    {
+                      identifier: 'maleCases',
+                      value: (0.5 + Math.random()),
+                    },
+                    {
+                      identifier: 'femaleCases',
+                      value: (0.5 + Math.random()),
+                    }
+                  ],
+                })),
+            })),
+        },
+        status: 200,
+      }));
+    }
+
+    if (request.url.includes('/distribution/age-sex/cured')) {
```

Comment:
if this is needed we need to add it to the apiary file

Reply:
At least it was written in my issue
I will add it to the file